recipe/package_data.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/setup.py b/setup.py
+index 6021604..1ac1cdd 100644
+--- a/setup.py
++++ b/setup.py
+@@ -43,6 +43,21 @@ problem before tyring to use the epics package.
+ """
+ 
+ 
++# determine which libca / libCom dll is appropriate
++import platform
++mach = platform.machine()
++if mach.startswith('arm'):
++    nbits = 'arm'
++else:
++    nbits = platform.architecture()[0].replace('bit', '')
++if os.name == 'nt':
++    libsrc = 'win'
++elif sys.platform == 'darwin':
++    libsrc = 'darwin'
++elif sys.platform.startswith('linux'):
++    libsrc = 'linux'
```

Comment:
It uses two different ways to get the platform. Can it be handled via the earlier used `platform`, such as [`platform.system()`](https://docs.python.org/3/library/platform.html#platform.system), that returns one of 'Linux', 'Darwin', 'Java', 'Windows' (needs to be lower-cased and the `win` case has to be handled specially)? Just not very happy with the `os`, `sys`, and `platform` used to determine it.

Reply:
I copied this code essentially verbatim from the library itself, but have updated the patch in line with your comments, please rereview.

Reply:
I do not see the changes here. Was the commit puahed?

Reply:
No. No it wasn't, sigh; sorry. It is there now.